### DISCUSSION TITLE
Siren never sound

### DISCUSSION THREAD

**Discussion by jolly12f** [ParadoxAlarmInterface/pai#428] [Q&A]
ip150 magellan 5050 my siren and my keypad never sound when there is an alarm, only when I am connected via PAI does this happen. if I use the swan system everything works

**Comment by yozik04** [comment]
Rephrase please. From what I understand and I can recommend is that you might have some special settings for System arming. Unfortunatelly I won't be able to help. Panel's manual is your friend =)

**Comment by jolly12f** [comment]
when my anti-theft system goes into alarm it doesn't make the siren sound since I'm connected to pai, but if I disconnect pai it works
